cypress/integration/participantView.spec.js

```diff
@@ -11,8 +11,16 @@ describe("Participant View", () => {
     cy.get('ul.MuiMenu-list[role=listbox] > li').should('have.length', 3);
 
     // Testing for freshness
-    cy.contains('a day ago').should('exist');
-    cy.contains('14 days ago').should('exist');
+
+    cy.get('body').then(($body) => {
+      if (Cypress.$('body:contains("a day ago")').length !== 0) {
```

Comment:
The response of moment's .fromNow() function fluctuates by a day based on when the test is run. This should cover both cases. I don't want to spend much more time on writing this test but if it seems important to figure out exactly where the discrepancy is coming from then I'll dig deeper.

Reply:
Great [lampshading](https://tvtropes.org/pmwiki/pmwiki.php/Main/LampshadeHanging).

Reply:
I bet this is down to the time of the server used by GHA when running the tests (or something like that)